src/ZendeskApi.Client/IZendeskClientFactory.cs

```diff
@@ -1,12 +0,0 @@
-﻿using System;
-using ZendeskApi.Client.Http;
-using ZendeskApi.Client.Logging;
-using ZendeskApi.Client.Serialization;
-
-namespace ZendeskApi.Client
-{
-    public interface IZendeskClientFactory
-    {
-        IZendeskClient Create(Uri baseUri, ZendeskDefaultConfiguration configuration, ISerializer serializer = null, IHttpChannel httpChannel = null, ILogAdapter logger = null);
```

Comment:
The only thing I see of use on this API is 'maybe' ISerializer... which maybe we could introduce back in to IOptions interface.

The rest I feel is covered by the Options abstraction